### ISSUE TITLE
Initializing image crop with a normalized crop CGRect

### ISSUE THREAD

**Issue by alessiolapenna** [guoyingtao/Mantis#56]
I have a different tool that as output provides a normalized CGRect as the crop of interest.
I'd like to use this crop to inizialize the CropViewController so that it can "start" with the crop I provide.
I cannot find an easy way to do this.
What is the correct way to achieve this? 
iOS interfaces work with normalized crop rects (in the range 0..1) so I think it would be a good improvement to add the possibility to set one in the framework.

**Comment by guoyingtao** [comment]
@alessiolapenna  Just want to make sure my understanding is right.

The original image
888888
888888
888888
888888
888888
888888

The iniialized image with the input crop information (0.6, 0.5)
 8888
 8888
 8888

**Comment by alessiolapenna** [comment]
@guoyingtao Thanks for your reply.
I mean I would give as an input cgrect(x:0.1, y:0.1, width:0.8, height:0.8) meaning that the origin should be at 10% of the size of the image and the size is 80%. 
So for example an image of size 200x200, passing the above rect I would get an image of 160x160 with origin at 20,20
I don't know if I could explain myself

**Comment by guoyingtao** [comment]
@alessiolapenna I think I understand what you mean. I can add this feature this weekend if I get enough time.

**Comment by alessiolapenna** [comment]
That would be great.
Thank you

Inviato da iPhone

**Comment by guoyingtao** [comment]
@alessiolapenna I added the new feature on 56_normailized_input branch by adding presetNormalizedInfo to PresetTransformationType
You can test it by setting config.presetTransformationType like this
```Swift
var config = Mantis.Config()
config.presetTransformationType = .presetNormalizedInfo(normailizedInfo: CGRect(x: 0.2, y: 0.5, width: 0.6, height: 0.3))
```

**Comment by alessiolapenna** [comment]
Thank you very much.
I’ll let you know how my tests go.

**Comment by alessiolapenna** [comment]
@guoyingtao this works very well. I didn't find any issue. 
Thank you very much for the hard work.

**Comment by guoyingtao** [comment]
@alessiolapenna Thanks for the test. I released Mantis 1.4.3 which includes this feature.
